packages/esm-providers-app/src/workspace/provider-form.workspace.tsx

```diff
@@ -350,22 +364,22 @@ const ProviderForm: React.FC<ProvideModalProps> = ({ closeWorkspace, provider, u
         <Column>
           <span className={styles.form__gender}>{t('gender', 'Gender*')}</span>
           <Controller
-            name="gender"
             control={control}
+            name="gender"
             render={({ field }) => (
               <ContentSwitcher
-                id="form__content_switch"
-                selectedIndex={field.value === 'F' ? 1 : 0}
-                selectionMode="manual"
-                onChange={(event) => field.onChange(event.index === 0 ? 'M' : 'F')}>
+                selectedIndex={field.value == 'M' ? 0 : 1}
+                onChange={(value) => {
+                  let { index, name, text } = value;
+                  field.onChange(name);
+                }}>
```

Comment:
```suggestion
               onChange={({ name }) => field.onChange(name)}>
```
Since we aren't interested in the other properties i.e `index` and `text` we should just extract the value of name